qfieldsync/gui/cloud_projects_dialog.py

```diff
@@ -121,20 +121,6 @@ def __init__(
 
         self.update_welcome_label()
 
-        if self.network_manager.has_token():
-            self.show_projects()
-            self.show()
-            self.createButton.setEnabled(True)
-        else:
-            CloudLoginDialog.show_auth_dialog(
-                self.network_manager,
-                lambda: self.on_auth_accepted(),
-                lambda: self.close(),
-                parent=self,
-            )
-            self.hide()
-            self.createButton.setEnabled(False)
-
```

Comment:
Is it required to move this block more to the bottom?

Reply:
@suricactus , I wanted to call self.show() after as much of the UI was set up as possible. It however has to be before we create CloudCreateProjectWidget has that will also open the login dialog within the constructor.